src/VisualStudio/IntegrationTest/TestUtilities/InProcess/VisualStudioWorkspace_InProc.cs

```diff
@@ -81,5 +83,23 @@ public void CleanUpWaitingService()
 
                 GetWaitingService().EnableActiveTokenTracking(true);
             });
+
+        public void SetFeatureOption(string feature, string optionName, string language, string valueString)
```

Comment:
Do tests need to revert any option changes they make in order to avoid breaking the option assumptions made by other tests?